examples/01_random_field/09_fourier_trans.py

```diff
@@ -0,0 +1,45 @@
+"""
+Generating a Transformed Periodic Random Field
+----------------------------------------------
+
+Building on the precious example, we are now going to generate periodic
+spatial random fields with a transformation applied, resulting in a level set.
+"""
+
+import numpy as np
+
+import gstools as gs
+
+# We start off by defining the spatial grid.
+L = np.array((500, 500))
```

Comment:
Maybe use different periods for different directions in this example?

Reply:
Good idea, implemented.